arcaflow_plugin_kill_pod.py

```diff
@@ -205,12 +206,15 @@ def kill_pods(
                 new_watch_pods: typing.List[Pod] = []
                 for p in watch_pods:
                     try:
-                        read_pod = core_v1.read_namespaced_pod(p.name, p.namespace)
+                        read_pod = core_v1.read_namespaced_pod(p.name, p.namespace)                        
                         if read_pod.metadata.name != p.name:
                             return "error", PodErrorOutput(
                                 "Error retrieveing pod {}".format(p.name)
                             )
-                        new_watch_pods.append(p)
+                        if read_pod.status.phase=='Running':
+                            if read_pod.metadata.creation_timestamp.tzinfo > kill_time.tzinfo:
```

Comment:
will this work if we are killing multiple pods? how do we track the kill_time for each of the pods

Reply:
@paigerube14, yes, it will work when we are killing multiple pods, as kill_time is recording the time the particular pod was killed and the value of read_pod.metadata.creation_timestamp.tzinfo will always be more if the pod is killed successfully. So even though we will have three different values of read_pod.metadata.creation_timestamp.tzinfo, one for each pod, these will always be greater than kill_time, if they are killed successfully. 
This is like an additional check if we just want to kill one pod and check if it was killed successfully.